packages/framework/client-logger/app-insights-logger/README.md

```diff
@@ -0,0 +1,3 @@
+# @fluidframework/app-insights-logger
```

Comment:
Same comments as the other README, but this README also really needs usage documentation as well. E.g.,:
- What are the use cases for this? What does this solve for potential consumers?
- How does the user initialize it? Some code examples would be useful (We can also probably point to the sample app package as an example).
-

Reply:
Added a readme.